IndustryEssentials/ymir
# [backend] repo and model versions

# changes
* use YMIR_REPO_VERSION to verify sandbox version
* use YMIR_MODEL_VERSION to verify model version
* sandbox_updater only update repo structures now
* model_updater updates model packages when:
    * import models
    * training: prepare pretrained models
    * mining: prepare model
    * infer: prepare model

# how to test
* i 've tested the following cases:
    * 1.1.0 update to 2.1.0: to test sandbox updater and model updater
    * 2.0.2 update to 2.1.0: to test data compatibility
